### DISCUSSION TITLE
请问怎么通过好友请求呢

### DISCUSSION THREAD

**Discussion by Megghy** [Hoshikawa-Kaguya/Sora#54] [Q&A] [answered]
`FriendAddEventArgs` 和 `OnebotFriendAddEventArgs` 里似乎都没有提供flag

**Comment by Yukari316** [comment, marked as answer]
啊，这个是之前适配的时候忘记写进去的，将会在新版本中添加
感谢提醒
~~还有就是这几天在肝毕设，可能回消息不会太快~~

**Reply by Megghy** [reply]
收到收到, 祝毕设顺利捏

**Reply by Yukari316** [reply]
emmmm，之前因为协议忘的差不多了，但我看了一下onebot v11的文档，`notice|friend_add` 事件是不会提供 `flag` 的
提供 `flag` 参数的是 `request|friend` 事件，也就是需要好友请求事件

**Reply by Megghy** [reply]
草原来是用错事件了吗, 那不好意思添麻烦了

**Reply by Yukari316** [reply]
问题不大
